### PR TITLE
Static Site

### PR DESCRIPTION
**Pull request by BSick7**
This PR implements `nullstone push`, `nullstone deploy`, `nullstone launch` for `app/static-site` app pattern for `aws-s3-site`.

### CODE REVIEW

File: app/static-site/aws-s3/infra_config.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+	"github.com/aws/aws-sdk-go-v2/service/cloudfront"
+	cftypes "github.com/aws/aws-sdk-go-v2/service/cloudfront/types"
+	nsaws "gopkg.in/nullstone-io/nullstone.v0/aws"
+	"gopkg.in/nullstone-io/nullstone.v0/contracts/aws-s3-site"
+	"log"
+	"os"
+	"strings"
+)
+
+// InfraConfig provides a minimal understanding of the infrastructure provisioned for a module type=aws-fargate
```
Comment (ssickles) [inline app/static-site/aws-s3/infra_config.go:16]: this says fargate in the comment, I think you meant s3 (copy/paste error?)

**Review by ssickles — APPROVED** [review]
Looks good!

File: app/static-site/aws-s3/infra_config.go (hunk 1 above)
Comment (BSick7) [reply app/static-site/aws-s3/infra_config.go:16]: Good find

Comment (BSick7) [inline app/static-site/aws-s3/infra_config.go:16]:
```suggestion
// InfraConfig provides a minimal understanding of the infrastructure provisioned for a module type=aws-s3-site
```

**Pull request merged**